packages/lesswrong/server/callbacks/rateLimits.ts

```diff
@@ -174,16 +175,18 @@ export async function rateLimitGetPostSpecificCommentLimit(user: DbUser, postId:
   nextEligible: Date,
   rateLimitType: RateLimitReason,
 }|null> {
-  if (postId && await userHasActiveModeratorActionOfType(user, RATE_LIMIT_THREE_COMMENTS_PER_POST)) {
+  if (postId && await userHasActiveModeratorActionOfType(user, RATE_LIMIT_THREE_COMMENTS_PER_POST_PER_WEEK)) {
+    const HOURS = 24 * 7
+    const NUMBER_OF_COMMENTS = 3
```

Comment:
[nit] Style comment, these shouldn't be capitalized unless they're global constants.